src/Views/Form/Themes/Sequoia/assets/js/form.js

```diff
@@ -1,39 +1,175 @@
 
-			// Show choose amount section
-			$( '.give-total-wrap', $container ).addClass( 'give-flex' );
-			$( '.give-donation-levels-wrap', $container ).addClass( 'give-grid' );
-			$( '.give-section.choose-amount', $form ).show();
+			const hide = steps.map( ( obj, index ) => {
+				if ( index !== step ) {
+					return obj.selector;
+				}
+			} );
+			const hideSelector = hide.filter( Boolean ).join( ', ' );
 
-			$parent.removeClass( 'give-showing__introduction-section' ).addClass( 'give-showing_choose-amount-section' );
-		} else if ( $parent.hasClass( 'give-showing_choose-amount-section' ) ) {
-			// Hide choose amount section.
-			$( '.give-total-wrap', $container ).removeClass( 'give-flex' );
-			$( '.give-section.choose-amount', $form ).hide();
+			$( hideSelector ).hide();
+			$( steps[ step ].selector ).show();
 
-			// Hide paginate button.
-			$( '.give-show-form', $container ).hide();
+			if ( step === steps.length - 1 ) {
+				$advanceButton.hide();
+			} else {
+				$advanceButton.show();
+			}
 
-			// Show remain form options.
-			$( 'form > *:not(.give-section.choose-amount)', $container ).show();
+			steps[ step ].setup();
+			navigator.currentStep = step;
+		},
+		back: () => {
+			const prevStep = navigator.currentStep !== 0 ? navigator.currentStep - 1 : 0;
+			navigator.goToStep( prevStep );
+			navigator.currentStep = prevStep;
+		},
+		forward: () => {
+			const nextStep = navigator.currentStep !== null ? navigator.currentStep + 1 : 1;
+			navigator.goToStep( nextStep );
+			navigator.currentStep = nextStep;
+		},
+	};
 
-			$parent.removeClass( 'give-showing_choose-amount-section' ).addClass( 'give-showing__personal-section' );
-		}
+	const steps = [
+		{
+			id: 'introduction',
+			title: 'Introduction',
+			selector: '.give-section.introduction, .give-section.income-stats, .give-section.progress-bar',
+			label: templateOptions.introduction.donate_label,
+			setup: () => {
+
+			},
+		},
+		{
+			id: 'choose-amount',
+			title: 'Choose Amount',
+			selector: '.give-section.choose-amount',
+			label: templateOptions.payment_amount.next_label,
+			setup: () => {
+				$( '.give-donation-level-btn' ).each( function() {
+					const hasTooltip = $( this ).attr( 'has-tooltip' );
+					if ( hasTooltip ) {
+						return;
+					}
+
+					const value = $( this ).attr( 'value' );
+					const text = $( this ).text();
+					if ( value !== 'custom' ) {
+						const wrap = `<span class="give-tooltip hint--top hint--bounce" style="width: 100%" aria-label="${ text }" rel="tooltip"></span>`;
+						const html = `<div class="currency">$</div>${ value }`;
+						$( this ).html( html );
+						$( this ).wrap( wrap );
+						$( this ).attr( 'has-tooltip', true );
+					}
+				} );
+				$( '.give-total-wrap', $container ).addClass( 'give-flex' );
+				$( '.give-donation-levels-wrap', $container ).addClass( 'give-grid' );
+			},
+		},
+		{
+			id: 'personal',
+			title: 'Add Your Information',
+			label: 'Process Donation',
```

Comment:
I noticed that you made `label` customizable for other buttons, can we do the same here?